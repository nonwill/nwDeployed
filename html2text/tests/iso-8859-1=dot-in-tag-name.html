<html>
<body>
<table width='620' border='0'>
<tr>
<td>
<pre>

hey,

The following backport has been assigned to 
you.

<p style="background-image: url(">See TOC below.&#160; &#160;</p><p
style="background-image: url(">https://somehost/Backport+Instructions&#160; &#160;</p><p style="background-image: url(">&#160;NOTE:&#160; &#160;</p><p style="background-image: url(">&#160;Please do the backports on the latest code&#160;&#160;</p><p style="background-image: url(">&#160;in git using the&#160; -bl switch:</p><p style="background-image: url(">&#160;"backport doit -backport&#160;<a_number>-bl&#160;<ab.cdef label="">"&#160; &#160;</ab.cdef></a_number></p><p style="background-image: url("><a_number>Only working content is acceptable.</a_number></p>
</pre>
</td>
</tr>
</table>
<br/>
</body>

</html>
