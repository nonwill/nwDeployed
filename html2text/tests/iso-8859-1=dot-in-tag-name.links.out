iso-8859-1=dot-in-tag-name.default.out